### DISCUSSION TITLE
can Vanna AI connect to a Power BI model?

### DISCUSSION THREAD

**Discussion by razakpm** [vanna-ai/vanna#644] [Q&A]
I have a requirement to connect Vanna.ai to a PowerBI model. Is there any option to connect to an already created PowerBI model? Or is there any other methodology supported by Vanna.ai to leverage the full capabilities of PowerBI for displaying dashboards?

**Comment by andreped** [comment]
Hello, @razakpm! :]

As far as I am aware, there is no easy way to do this. I have worked with PowerBI a bit before, and allowing to connect an external API from a python backend can be quite tricky. I would at least expect there to be quite some overhead, but can't say for certain.

Perhaps this is something you have tried before, @zainhoda, or someone else maybe? Would be a really cool project to demo such a feature though 😎
